### PR TITLE
Hiya

### PR DESCRIPTION
**Pull request by hg01195**
@nocapsnospaces  add members to team, all code moved to route, corrections from previous PR made. Please merge when possible.

### CODE REVIEW

File: athtrack/models.py [hunk 1]
```diff
@@ -90,4 +91,3 @@ class SurveyOptions(db.Model):
 class SurveyAnswers(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     choice_id = db.Column(db.Integer, db.ForeignKey('survey_options.id'), nullable=False)
-    student = db.Column(db.Integer, db.ForeignKey('athlete.id'), nullable=False)
```
Comment (nocapsnospaces) [inline athtrack/models.py:93]: Why did you kill this line? We have to record survey answers, right?

File: athtrack/routes.py [hunk 2]
```diff
@@ -1,94 +1,127 @@
-import os
```
Comment (nocapsnospaces) [inline athtrack/routes.py:1]: Did you strip or add CR/LF here? was it giving you issues?

File: athtrack/routes.py [hunk 3]
```diff
@@ -1,94 +1,127 @@
...
+        Expects
+        ```json
+        {"students":[ 7,8,<student_id>, <student_id>]}
+        ```
+        """
+    errors = []
+    if not request.is_json:
+        return Response(json.dumps({'msg': "Bad request MIME type"}), status=400)
+    dictionary = request.get_json()
+    students = dictionary['students']
```
Comment (nocapsnospaces) [inline athtrack/routes.py:111]: This line will raise an exception if 'students' is not present. You must either surround in a try/catch or use `students = dictionary.get('students', None)`

File: migrations/alembic.ini [hunk 4]
```diff
@@ -1,45 +0,0 @@
-# A generic, single database configuration.
```
Comment (nocapsnospaces) [inline migrations/alembic.ini:1]: Why did you delete these files? This is an error.

File: migrations/README [hunk 5]
```diff
@@ -1 +0,0 @@
-Generic single-database configuration.
```
Comment (nocapsnospaces) [inline migrations/README:1]: Again, why deleted?

File: migrations/env.py [hunk 6]
```diff
@@ -1,97 +0,0 @@
-from __future__ import with_statement
```
Comment (nocapsnospaces) [inline migrations/env.py:1]: we cannot delete this file

File: migrations/script.py.mako [hunk 7]
```diff
@@ -1,24 +0,0 @@
-"""${message}
```
Comment (nocapsnospaces) [inline migrations/script.py.mako:1]: we cannot delete this file

File: athtrack/models.py (hunk 1 above)
Comment (nocapsnospaces) [reply athtrack/models.py:93]: Looking at the migration you added, I think this was an error.

**Comment by hg01195** [comment]
merging nightmare, closing and raising diff PR
